src/Microsoft.ML.Data/Training/TrainContext.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Runtime.Training
+{
+    /// <summary>
+    /// A training context is an object instantiable by a user to do various tasks relating to a particular
```

Comment:
>context [](start = 19, length = 7)

would "ml task" sound more familiar?we're overloading TrainContext a bit.

Reply:
'Task' is a term we'd rather leave for .NET's normal use, which is for parallelization

---
In reply to: [219307766](https://github.com/dotnet/machinelearning/pull/967#discussion_r219307766) [](ancestors = 219307766)